Source/UI/NewUI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -214,11 +215,17 @@ extension DefaultBillingFormViewModel: BillingFormTextFieldDelegate {
 // MARK: - Billing form view controller Delegate
 
 extension DefaultBillingFormViewModel: BillingFormViewControllerDelegate {
+    func phoneNumberIsUpdated(number: String?) {
```

Comment:
Looking at the code changes, inside `TextFieldView` there was a TODO, asking to validate the number. That was now removed, with the code simply sending the message via delegate back here (I think as not very clear). Now there is still no validation being done.

Are we happy that is no longer needed?